integration-tests/src/test/resources/features/filter.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Filter tests
```

Comment:
@Aakanksha0311 Please put the test into `processor-filter.feature` to have filtering tests in one place.